src/sentry/tasks/digests.py

```diff
@@ -73,6 +76,7 @@ def deliver_digest(key, schedule_timestamp=None, notification_uuid: Optional[str
                 target_type,
                 target_identifier,
                 fallthrough_choice=fallthrough_choice,
+                notification_uuid=notification_uuid,
```

Comment:
@scttcper does `notify_digest` have the `notification_uuid` argument already? What I was going to say it might be safer just to add the argument everywhere first then start passing it in. At least for anything that is a task.

Reply:
thats what i'm doing here https://github.com/getsentry/sentry/pull/56230